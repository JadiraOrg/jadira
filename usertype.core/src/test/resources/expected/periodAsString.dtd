<!ELEMENT dataset (
    PERIODASSTRING*)>

<!ELEMENT PERIODASSTRING EMPTY>
<!ATTLIST PERIODASSTRING
    ID CDATA #REQUIRED
    NAME CDATA #IMPLIED
    PERIOD CDATA #IMPLIED
>

