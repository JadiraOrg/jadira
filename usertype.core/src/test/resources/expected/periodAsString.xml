<?xml version='1.0' encoding='UTF-8'?>
<!--

  Copyright 2010, 2011 Christopher Pheby

  Licensed under the Apache License, Version 2.0 (the "License");
  you may not use this file except in compliance with the License.
  You may obtain a copy of the License at

      http://www.apache.org/licenses/LICENSE-2.0

  Unless required by applicable law or agreed to in writing, software
  distributed under the License is distributed on an "AS IS" BASIS,
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.

 -->
<dataset>
  <periodAsString ID="0" NAME="test_0" PERIOD="P3M"/>
  <periodAsString ID="1" NAME="test_1" PERIOD="P4Y35M40D"/>
  <periodAsString ID="2" NAME="test_2" PERIOD="P28Y10M2D"/>
  <periodAsString ID="3" NAME="test_3" PERIOD="P28Y10M"/>
</dataset>
